code/server/sv_ccmds.c

```diff
@@ -288,11 +288,6 @@ static void SV_MapRestart_f( void ) {
 	// map_restart has happened
 	svs.snapFlagServerBit ^= SNAPFLAG_SERVERCOUNT;
 
-	// generate a new serverid	
-	// TTimo - don't update restartedserverId there, otherwise we won't deal correctly with multiple map_restart
-	sv.serverId = com_frameTime;
-	Cvar_SetIntegerValue( "sv_serverid", sv.serverId );
-
```

Comment:
this will cause at least following condition to stop working:
https://github.com/ec-/Quake3e/pull/283/commits/bc0a9fea99d2c382163011207a36c5d32725d225#diff-7357892ed859d7961b2dce3cdfdc28519c4a3781a9e4a7431226171ebed64298R244-R247